{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -53,51 +51,52 @@ If you are proposing a feature:
 
 Ready to contribute? Here's how to set up **{{cookiecutter.project_name }}** for local development.
 
-1. Fork the **{{ cookiecutter.project_name }}** repo on {{ cookiecutter.vs | title }}.
-
-2. Clone your fork locally:
+1. Fork the **{{ cookiecutter.project_name }}** repository.
 
-        $ git clone ssh://git@{{ cookiecutter.vs_url }}{% if cookiecutter.vs|lower == "bitbucket" %}:7999/~{% else %}/{% endif %}your_name_here/{{ cookiecutter.project_name }}.git
+1. Clone your fork locally.
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenv installed,
+1. Install your local copy into a virtualenv. Assuming you have virtualenv installed,
    this is how you set up your fork for local development:
 
         $ cd {{ cookiecutter.project_name }}/
-        $ virtualenv env
+        $ virtualenv --python=python3 env
         $ source env/bin/activate
         $ python setup.py develop
 
-4. Create a branch for local development:
+1. Create a branch for local development and start making your changes locally:
 
         $ git checkout -b name-of-your-bugfix-or-feature
 
-  Now you can make your changes locally.
+1. When you're done making changes, check that your changes pass pylint and the tests:
 
-5. When you're done making changes, check that your changes pass pylint
-   and the tests, including testing other Python versions with tox:
+        $ python setup.py lint
+        $ python setup.py test
+
+{% if cookiecutter.use_tox|lower == "y" -%}
+
+    Use tox for running the validations against different Python versions:
 
         $ tox
 
-  To get tox, just pip install it into your virtualenv.
+    To get tox, just pip install it into your virtualenv.
```

Comment:
Isn't it in requirements_dev.txt when the tox option is selected? Maybe there should be a step to install all of those.

Reply:
Very good point. Will remove that part and add the pip install command